GiEnJul/Controllers/AdminController.cs

```diff
@@ -240,6 +240,25 @@ public async Task<ActionResult> DeleteRecipientAsync([FromBody] DeleteRecipientD
             return Ok();
         }
 
+        [HttpPut("recipient")]
+        [Authorize(Policy = "UpdateRecipient")]
+        public async Task<ActionResult> PutRecipientAsync([FromBody] PutRecipientDto recipientDto)
+        {
+            var recipientToPut = _mapper.Map<Recipient>(recipientDto);
+            var recipientToSet = await _recipientRepository.GetRecipientAsync(recipientDto.PartitionKey, recipientDto.RowKey);
```

Comment:
Naming suggestion:
recipientToPut -> recipientNew, updatedRecipient, or keep as is
recipientToSet -> recipientOld , recipientToUpdate

Reply:
recipientToPut is now recipientUpdated
recipientToSet is now recipientToUpdate